### PR TITLE
Logan Ralston

### PR DESCRIPTION
**Pull request by logan-r**
submitting assignment 2

### CODE REVIEW

**Comment by jenny-xly-long** [comment]
Good work apart that you didn't do the last question. Here are some comments:
1. For the question of degree 2, 10 and 20. When the fit is of degree 2, the MSE is high due to underfitting, the model is not complex enough to fit the data. And when it is of degree 10, the MSE decreased because the model can better capture the data and when it is of degree 20, the MSE increased drastically because of overfitting.
2. I have attached a file of gradient descent algorithm that I've used in Comp 551 (I used the vector approach), hope it helps. :)
